### DISCUSSION TITLE
after updating the wallet to version 1.7.0, I can't log in using the keyword

### DISCUSSION THREAD

**Discussion by lubarets-evgeni** [Chia-Network/chia-blockchain#14825] [Support]
After updating the wallet to version 1.7.0, I can't log in using the keyword. a couple of months ago, I calmly entered the wallet using my SID phrase, then I did not use it and did not delete it, the other day I went in and the wallet requested the sid phrase - after entering a message that the sid phrase is incorrect (which cannot be, it is written only on paper and has been entered correctly on the same device several times). When entering this phrase into other web wallets, it is accepted to the coin balance shows 0. Please help restore the wallet.

**Comment by loppefaaret** [comment]
duplicate of #14826 
Started in above issue
